browser/src/control/jsdialog/Widget.Progressbar.js

```diff
@@ -42,6 +42,11 @@ JSDialog.progressbar = function (parentContainer, data, builder) {
 	else
 		progressbar.max = 100;
 
+	if (data.infinite === true) {
```

Comment:
Please add condition: if value is already 100% -> do not add `infinite` class.

So simply when completed -> finish animation :)